proto/ligato/vpp/l3/vrrp.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package ligato.vpp.l3;
+
+option go_package = "go.ligato.io/vpp-agent/v3/proto/ligato/vpp/l3;vpp_l3";
+
+message VRRPEntry {
+    string interface = 1;
+    uint32 vr_id = 2;
+    uint32 priority = 3;
+    // VR advertisement interval.
+    uint32 interval = 4;
+    // Controls whether a (starting or restarting) 
+    // higher-priority Backup router preempts a lower-priority Master router.
+    bool preempt = 5;
+    // Controls whether a virtual router in Master state will accept packets
+    // addressed to the address owner's IPvX address as its own if it is not the IPvX address owner.
+    bool accept = 6;
+    // Unicast mode may be used to take
+    // advantage of newer token ring adapter implementations that support
+    // non-promiscuous reception for multiple unicast MAC addresses and to
+    // avoid both the multicast traffic and usage conflicts associated with
+    // the use of token ring functional addresses.
+    bool unicast = 7;
+    bool ipv6 = 8;
+
+    repeated string addrs = 9;
```

Comment:
If these are IP addresses it would be better named `ip_adresses` to be more consistent and clear to user.